bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioServlet.java

```diff
@@ -37,7 +39,7 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * A servlet that serves audio streams via HTTP.
+ * A servlet that serves audio oneTimeStreams via HTTP.
```

Comment:
We should keep "streams" here. I assume this has been done by the refactor procedure.